src/main/java/com/paloaltonetworks/osc/api/PANManagerPolicyApi.java

```diff
@@ -30,35 +30,36 @@
  */
 public class PANManagerPolicyApi implements ManagerPolicyApi {
 
-    private static final Logger log = Logger.getLogger(PANManagerPolicyApi.class);
-    static String apiKey = null;
-    private VirtualSystemElement vs;
-    private ApplianceManagerConnectorElement mc;
-    private ShowOperations showOperations;
+	private static final Logger log = Logger.getLogger(PANManagerPolicyApi.class);
+	static String apiKey = null;
+	private VirtualSystemElement vs;
+	private ApplianceManagerConnectorElement mc;
+	private ShowOperations showOperations;
 
-    private static ArrayList<PolicyListElement> policyList = new ArrayList<>();
-    static {
-        policyList.add(new PolicyListElement("Platinum", "Platinum"));
-        policyList.add(new PolicyListElement("Gold", "Gold"));
-        policyList.add(new PolicyListElement("Silver", "Silver"));
-        policyList.add(new PolicyListElement("Bronze", "Bronze"));
-    }
+	private static ArrayList<PolicyListElement> policyList = new ArrayList<>();
+	static {
+		// Add domain if applicable
+		policyList.add(new PolicyListElement("Platinum", "Platinum", null));
```

Comment:
pan generally reference Root-Domain. see PANManagerDomainApi. Use that instead of null